### PR TITLE
Feat cookbook recipe articles

### PR DESCRIPTION
**Pull request by MOshima-PIFSC**
add vignettes for documentation to gh pages site

### CODE REVIEW

File: .github/workflows/R-CMD-check.yaml
Comment (efletcherPIFSC) [inline .github/workflows/R-CMD-check.yaml:1]: Remove branches-ignore: EAT-cookbook-recipe-articles

File: .github/workflows/pkgdown.yaml
Comment (efletcherPIFSC) [inline .github/workflows/pkgdown.yaml:1]: Remove FEAT-cookbook-recipe-articles

File: Vignette/ss3diags_handbook.Rmd
Comment (efletcherPIFSC) [inline Vignette/ss3diags_handbook.Rmd:1]: README code moved to vignettes

File: data/mcmcSimple.rda
Comment (efletcherPIFSC) [inline data/mcmcSimple.rda:1]: data files (R datafiles and SS model files) now generated by vignettes via r4ss

File: vignettes/.gitignore
Comment (efletcherPIFSC) [inline vignettes/.gitignore:1]: Converted model recipe scripts to Vignettes

**Review by efletcherPIFSC — COMMENTED** [review]
Remove branches-ignore: EAT-cookbook-recipe-articles

Other Reviewer Comments noted for changelog

File: .github/workflows/pkgdown.yaml
Comment (MOshima-PIFSC) [reply .github/workflows/pkgdown.yaml:1]: Done

**Comment by MOshima-PIFSC** [comment]
This PR addresses issues #78, #75, #73, and #68.

**Pull request merged**